README.md

```diff
@@ -14,21 +14,23 @@ Project | Language | Visibility | Description
 --- | --- | --- | ---
 [Client Template](https://github.com/LaunchPadLab/client-template) | JS, React, Redux | Private | Template app for React/Redux SPAs.
 [Decanter](https://github.com/LaunchPadLab/decanter) | Ruby, Rails | Public | Gem that makes it easy to transform incoming data before it hits the controller action or model.
-[Fuel](https://github.com/LaunchPadLab/fuel) | Ruby, Rails | Public | Stop developing Rails blogs and start writing your actual blog posts with this dead simple blogging engine for Rails.
+[Flight Readiness Review](https://github.com/LaunchPadLab/flight_readiness_review) | Bash | Private | Utility to set up a macOS laptop for design and development at LaunchPad Lab.
+[Fuel](https://github.com/LaunchPadLab/fuel) (no longer maintained) | Ruby, Rails | Public | Stop developing Rails blogs and start writing your actual blog posts with this dead simple blogging engine for Rails.
 [LP Components](https://github.com/LaunchPadLab/lp-components) | JS, React | Public | A set of reusable React components.
 [LP Form](https://github.com/LaunchPadLab/lp-form) | JS, Redux | Public | Extensions for the reduxForm HOC.
-[LP HOC](https://github.com/LaunchPadLab/lp-hoc) | JS, React | Public | A set of higher order components (HOC) for use in React apps.
+[LP HOC](https://github.com/LaunchPadLab/lp-hoc) (deprecated) | JS, React | Public | A set of higher order components (HOC) for use in React apps.
```

Comment:
Do we want to note that `hooks` is the replacement?

Reply:
We could make the language stronger / change the sequencing in the LP HOC library? https://github.com/LaunchPadLab/lp-hoc#a-note-about-hooks

Reply:
I think what we have works for now!